src/database.ts

```diff
@@ -1994,11 +2057,18 @@ class Database extends common.GrpcServiceObject {
   getTransaction(
     callback?: GetTransactionCallback
   ): void | Promise<[Transaction]> {
-    this.pool_.getSession((err, session, transaction) => {
+    this.pool_.getSession(false, (err, session, transaction) => {
       if (!err) {
         this._releaseOnEnd(session!, transaction!);
+        session!.txn = transaction;
+      }
+      if (
```

Comment:
This is because typescripts throws an error now that transaction can be of snapshot type as well: https://github.com/googleapis/nodejs-spanner/pull/1863/files/141b8e9140697c987b4b9ec0438fedbb11baa5e6#diff-96998a2148aa3d52ad3e80303d0ad88fc43a4afd08966b5330a5657021ae52bbR117